packages/cli/src/commands/build-docs/types.ts

```diff
@@ -19,5 +19,8 @@ export type BuildDocsArgv = {
   disableGoogleFont?: boolean;
   template?: string;
   templateOptions: Record<string, any>;
-  options: string | Record<string, unknown>;
+  features: {
+    openapi: string | Record<string, unknown>;
+  };
```

Comment:
Isn't `features.openapi` the single string? Not an object in an object?

Reply:
No, it workaround flow for support `dot-notation`.  I tried update to the latest version of yargs, but they still have this issue.
```bash
redocly build-docs -- --config=./resources/.redocly.yaml --features.openapi.theme.colors.primary.main=red
```
<img width="693" alt="Screenshot 2022-09-26 at 14 48 55" src="https://user-images.githubusercontent.com/14113673/192269051-6ca00246-8532-4e94-8497-a0eabc219091.png">